# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/nsCSSValue.h
# Commit: 9525c5d9a710
# Full Hash: 9525c5d9a710cd1a2c960389c03a31aba9e02003
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-13 09:25:56
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 2 - support vw, vh, vmin and vmax units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180619
# ==============================================================================

diff -r a16d763f2866 -r 9525c5d9a710 layout/style/nsCSSValue.h
--- a/layout/style/nsCSSValue.h	Tue Sep 12 21:10:54 2023 +0000
+++ b/layout/style/nsCSSValue.h	Tue Sep 12 21:10:55 2023 +0000
@@ -55,12 +55,12 @@
                            // different behavior than percent)
 
   // Font relative measure
-  eCSSUnit_EM = 800,       // == current font size
-  eCSSUnit_XHeight = 801,  // distance from top of lower case x to
-                           // baseline
-  eCSSUnit_Char = 802,     // number of characters, used for width with
-                           // monospace font
-  eCSSUnit_RootEM = 803,   // == root element font size
+  eCSSUnit_EM = 800,           // == current font size
+  eCSSUnit_XHeight = 801,      // distance from top of lower case x to
+                               // baseline
+  eCSSUnit_Char = 802,         // number of characters, used for width with
+                               // monospace font
+  eCSSUnit_RootEM = 803,       // == root element font size
   eCSSUnit_Ideographic = 804,  // == CJK water ideograph width
   eCSSUnit_CapHeight = 805,    // == Capital letter height
 
@@ -72,6 +72,12 @@
   eCSSUnit_Pica = 904,        // 12 points == 16 CSS pixls
   eCSSUnit_Quarter = 905,     // 96/101.6 CSS pixels
   eCSSUnit_Pixel = 906,       // CSS pixel unit
+
+  // Viewport percentage lengths
+  eCSSUnit_VW = 950,
+  eCSSUnit_VH = 951,
+  eCSSUnit_VMin = 952,
+  eCSSUnit_VMax = 953,
 };
 
 struct nsCSSValuePair;